src/jit/lsraarm.cpp

```diff
@@ -454,103 +489,22 @@ void Lowering::TreeNodeInfoInitCall(GenTreeCall* call)
             continue;
         }
 
-        var_types argType    = argNode->TypeGet();
-        bool      argIsFloat = varTypeIsFloating(argType);
-        callHasFloatRegArgs |= argIsFloat;
-
-        regNumber argReg = curArgTabEntry->regNum;
-        // We will setup argMask to the set of all registers that compose this argument
-        regMaskTP argMask = 0;
```

Comment:
I note that identical code also appears in lsraarm64.cpp (line 980)

Apparently we compute argMask but it is never used/needed.